tests/bluetooth/bsim/host/adv/encrypted/css_sample_data/src/common.c

```diff
@@ -0,0 +1,36 @@
+/* Copyright (c) 2023 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common.h"
+
+#include "bs_cmd_line.h"
+
+const struct test_sample_data *sample_data;
+
+int data_set;
+
+void test_args_parse(int argc, char *argv[])
+{
+	bs_args_struct_t args_struct[] = {
+		{
+			.dest = &data_set,
+			.type = 'i',
+			.name = "{1, 2}",
+			.option = "data-set",
+			.descript = "Sample data set ID",
+		},
+	};
+
+	bs_args_parse_all_cmd_line(argc, argv, args_struct);
+
+	if (data_set != 1 && data_set != 2) {
+		data_set = 1;
+	}
+
+	if (data_set == 1) {
+		sample_data = &sample_data_1;
+	} else if (data_set == 2) {
+		sample_data = &sample_data_2;
+	}
```

Comment:
What happens if someone sets `data_set = 3`?

Reply:
why would someone do that? :p
I think more bsim tests (and tests in general) are not coded to be up to the same standards as applications, as they are not intended to be used by end-users.

anyway you have your answer a few lines above in this case.

Reply:
Indeed, if `data_set` is different from 1 or 2, it will default to 1.